### ISSUE TITLE
Creating axios instance doesn't append specified config params

### ISSUE THREAD

**Issue by marcus-sa** [axios/axios#1616]
#### Summary
_I'm using `ts-jest` for unit-testing, which means I don't know if the error still occurs outside the `jest` environment._

Basically I'm creating a new instance using the script below.
9/10 times the params doesn't get appended to the url.
```javascript
this.api = axios.create({
	baseURL: 'https://api.themoviedb.org/3',
	params: {
		api_key: 'api key goes here',
		language: 'da-DK',
		append_to_response: 'external_ids,videos'
	}
});
```

Using the request interceptor I logged the config output which was this:

```
{ adapter: [Function: httpAdapter],
        transformRequest: { '0': [Function: transformRequest] },
        transformResponse: { '0': [Function: transformResponse] },
        timeout: 0,
        xsrfCookieName: 'XSRF-TOKEN',
        xsrfHeaderName: 'X-XSRF-TOKEN',
        maxContentLength: -1,
        validateStatus: [Function: validateStatus],
        headers: 
         { common: { Accept: 'application/json, text/plain, */*' },
           delete: {},
           get: {},
           head: {},
           post: { 'Content-Type': 'application/x-www-form-urlencoded' },
           put: { 'Content-Type': 'application/x-www-form-urlencoded' },
           patch: { 'Content-Type': 'application/x-www-form-urlencoded' } },
        method: 'get',
        baseURL: 'https://api.themoviedb.org/3',
        params: undefined,
        url: 'movie/427641' }
```

and as you can see, the params are undefined.

For now I've managed to workaround it by defining the params in the interceptor:
```javascript
this.api.interceptors.request.use(config => ({
	...config,
	params: { 
		// ...
	},
}));
```

#### Context

- axios version: latest
- Environment: node v9.11.1

**Comment by danielfranca-ene** [comment]
Having the same issue but with 0.19 beta, 0.18 works.

**Comment by gecko25** [comment]
same^^ 0.19 not working, 0.18 is.

**Comment by cainster** [comment]
tried changing dependency to 0.18 in package.json but still not working for me ? Am I missing something?

**Comment by halfzebra** [comment]
The same issue, it seems like there's a PR with a fix #2196

**Comment by smac89** [comment]
Hey guys, for now you can just configure your interceptor to add the params.
```typescript
axios.interceptors.request.use((config: AxiosRequestConfig) => {
    config.params = {
        apiKey: "foobar"
    };
    return config;
});
```

**Comment by chinesedfan** [comment]
Closed in favor of #2190.
